src/components/shared/overlay/Overlay.tsx

```diff
@@ -34,7 +34,12 @@ class Overlay extends React.Component<IOverlayProps, IOverlayState> {
     const { opacity } = this.props;
 
     return (
-      <div className="overlay" onClick={this.onClick} style={{ opacity }} />
+      <div
+        className="overlay"
+        onClick={this.onClick}
+        style={{ opacity }}
+        data-testid="overlay"
```

Comment:
I think @HorusGoul mentioned in discord that if we added `data-testid` attributes that we should look into stripping them out of the production build.  Perhaps the `container.querySelector(".overlay")` selector is easier for now

Reply:
Found it https://discord.com/channels/808364903822917662/808364903822917665/814833758884200449

Reply:
Okay might have overlooked that one.

If we are using babel we can use this plugin; 
https://github.com/coderas/babel-plugin-jsx-remove-data-test-id

Let me know if you'd want me to remove the testId or add the babel plugin :)

Reply:
We can add the plugin later